pkgs/applications/networking/cluster/k3s/README.md

```diff
@@ -9,6 +9,7 @@ K3s is a simplified [Kubernetes](https://wiki.nixos.org/wiki/Kubernetes) version
 ## Configuration Examples
 
 * [Nvidia GPU Passthru](docs/examples/NVIDIA.md)
+* [Intel Discrete GPU Passthru](docs/examples/INTEL.md)
```

Comment:
We can probably just change this to `Intel GPU Passthru` as the steps you describe also work for iGPU passthru. I do the same on my N100 based mini-PCs for trans-coding. Possibly add a note to the examples doc that mentions this applies to both dGPU and iGPU but leave the rest the same for your dGPU example.

Reply:
Does this work? https://github.com/NixOS/nixpkgs/compare/64d37a09ba98176a256f41ddc594a6eb814df2bd..fbe127d6418372cfcfbd080fe0a2ecc0e7aeb626